Cargo.toml

```diff
@@ -1,9 +1,18 @@
 [package]
 name = "eslauncher2"
 version = "0.4.5"
-authors = ["M*C*O <mcofficer@gmx.de>"]
+authors = ["EndlessSkyCommunity"]
 edition = "2018"
 build = "build.rs"
+description = "Endless Sky Launcher"
```

Comment:
You can expand on this, e.g. "A plugin and game resource manager for Endless Sky"`

Reply:
Updated